lib/generators/pakyow/app/templates/app/setup.rb

```diff
@@ -13,6 +13,7 @@
 
     # put global config here and they'll be available across environments
     app.name = '<%= app_name %>'
+    logger.enabled = true
```

Comment:
TIL why `Gem#latest_version` is so slow -- thanks for taking the time to look into that!

Your suggestion of only looking at the local system sounds great to me. That will enable us to generate the appropriate version of the `Gemfile` and save us (and our users) some headaches.